tools/run_tests/artifacts/artifact_targets.py

```diff
@@ -433,8 +433,8 @@ def targets():
         CSharpExtArtifact('macos', 'ios', presubmit=True),
         PythonArtifact('manylinux2014', 'x64', 'cp36-cp36m', presubmit=True),
         PythonArtifact('manylinux2014', 'x64', 'cp37-cp37m'),
-        PythonArtifact('manylinux2014', 'x64', 'cp38-cp38'),
-        PythonArtifact('manylinux2014', 'x64', 'cp39-cp39'),
+        PythonArtifact('manylinux2014', 'x64', 'cp38-cp38', presubmit=True),
+        PythonArtifact('manylinux2014', 'x64', 'cp39-cp39', presubmit=True),
```

Comment:
Do we want to run this test per-commit or post-submit? If no flags is provided in the Kokoro cfg `tools/internal_ci/linux/grpc_gcp_distribtests_python.cfg`, then all artifacts will be built by default.

Reply:
This is my biggest undecided point about this. It would be _nice_ to know on a triggering PR whether or not it will break GCF, but we don't know yet how flaky it will be.

As discussed offline, we'll start this off as a presubmit test, since that gets us the most value. If flakiness becomes a problem, we'll move it to post-submit.

Reply:
Fair enough. Reverted the presubmit artifact changes, but leaving the `latest-manylinux` addition, since we'll need that functionality regardless.